plugins/modules/cloudformation_info.py

```diff
@@ -81,7 +81,7 @@
 # in ansible_facts['cloudformation'][<stack_name>] and can be used as follows.
 # Note that this is deprecated and will stop working in Ansible 2.13.
 
-- cloudformation_facts:
+- amazon.aws.cloudformation_facts:
```

Comment:
@tremble We talked about a similar example in the `community.aws` PR and because how the module is called affects the return and is noted as such in the comment I think it makes sense to keep this example.